solr/src/main/conf/schema.xml

```diff
@@ -0,0 +1,613 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
Solr does require a schema file that matches the data/fields intended to be
indexed/searched.

This schema file is being included for *example* purposes *only*, for the
sole purpose of illustrating by *example* what a schema file would look
like for the default fields used by the YCSB client. It is *not* expected
to be used for all possible benchmarking studies nor supported in any way.

Perhaps the location of this sample file is throwing people off??



On Wed, Jan 6, 2016 at 9:35 PM, Vitaliy Funshteyn <notifications@github.com>
wrote:

> In solr/src/main/conf/schema.xml
> <https://github.com/brianfrankcooper/YCSB/pull/574#discussion_r49012782>:
>
> > @@ -0,0 +1,613 @@
> > +<?xml version="1.0" encoding="UTF-8" ?>
> > +<!--
> > + Licensed to the Apache Software Foundation (ASF) under one or more
>
> The field name/quantity selection happens above the DB implementation
> layer (this code), namely in workload code. Why would we couple schema
> definition here to what the default workload does by default? Someone might
> create another workload config with a different field number - wouldn't
> this schema get out of sync?
>
> Also, does Solr really requires a fixed set of user-level field
> definitions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brianfrankcooper/YCSB/pull/574/files#r49012782>.
>